pages/getting-started/install-memgraph/kubernetes.mdx

```diff
@@ -21,32 +19,90 @@ There is also support for deploying Memgraph Lab on Kubernetes:
 - [**Helm chart for Memgraph Lab**](#helm-chart-for-memgraph-lab)
 
 For details on the implementation of the Helm charts, you can go to [**Memgraph
-Helm charts repository**](https://github.com/memgraph/helm-charts). Due to
-numerous possible use cases and deployment setups via Kubernetes, the provided
-Helm charts are a starting point you can modify according to your needs. 
+Helm charts repository**](https://github.com/memgraph/helm-charts).
+
+Due to numerous possible use cases and deployment setups via Kubernetes, the
+provided Helm charts are a starting point you can modify according to your
+needs. This page will highlight some of the specific parts of the Helm charts
+that you might want to adjust. 
 
-This docs page contains the basic information on how to deploy Memgraph or
-Memgraph Lab on Kubernetes using Helm charts.
 
 ## Helm chart for standalone Memgraph
 
-Since [Helm chart for standalone
+Memgraph is a stateful application (database), hence the [Helm chart for
+standalone
 Memgraph](https://github.com/memgraph/helm-charts/tree/main/charts/memgraph) is
-configured to deploy Memgraph as a Kubernetes `StatefulSet` workload, it is also
-necessary to define a `PersistentVolumeClaims` to store [the data
+configured to deploy Memgraph as a Kubernetes `StatefulSet` workload. 
+
+It will deploy a single Memgraph instance in a single pod.
+
+Typically, when deploying a stateful application like Memgraph, a `StatefulSet`
+workload is used to ensure that each pod has a unique identity and stable
+network identity. When deploying Memgraph,  it is also necessary to define a
+`PersistentVolumeClaims` to store [the data
 directory](/configuration/data-durability-and-backup) (`/var/lib/memgraph`).
 This enables the data to be persisted even if the pod is restarted or deleted. 
 
-If you don't require data persistence or your dataset is static, using the
-`StatefulSet` workload is unnecessary. Stateful applications are more complex to
-set up and maintain, as they require more attention when handling storage
-information and security.
+
+### Storage configuration
+
+By default, the Helm chart will create a `PersistentVolumeClaim` for storage and
+logs. The storage class is not defined by default, Memgraph will use the default
+storage class of the Kubernetes cluster. If your storage class does not have
+`Retain` reclaim policy, you will lose the data once the `PersistentVolumeClaim`
+is deleted because the data will be deleted with the `PersistentVolume`.
```

Comment:
This section is unclear with what's actually default and what's not. Please rewrite for clarity